tests/test_mdn.py

```diff
@@ -0,0 +1,111 @@
+# coding: utf-8
+
+import unittest
+import sys
+import os
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', 'nnsvs'))
+import mdn
```

Comment:
I think the first line is unnecessary. Please consider replacing the second line with `from nnsvs import mdn`

Reply:
I changed to use  predictin_type() of BaseModel in the latest commit.

Reply:
I changed import directive as you recommended in the latest commit.